source/_posts/akr-replica-setup.md

```diff
@@ -0,0 +1,151 @@
+---
+title: GCP MySQL replica setup
+date: 2020-09-14 19:06:35
+author: Abhishek Rana
+category: Web
+tags:
```

Comment:
keyword is not mentioned in the tag place

Reply:
I have added keywords from your list which were relevant to blog